source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -166,23 +168,34 @@ Install |CL| with Advanced Installation
 #. Complete the remaining steps of :guilabel:`Required Options` to 
    to install |CL|. Complete any :guilabel:`Advanced Options` as desired.
     
-Ways to boot |CL|
-*****************
+Three Methods to Boot |CL|
+*************(************
```

Comment:
Looks like you need to make all these asterisks. Minor typo.

Reply:
Hey @bktan8. Could you please make these changes. 
I tested this. Technically, it builds without errors. Syntax still needs to be fixed.